phoenix-core/src/main/java/org/apache/phoenix/hbase/index/IndexRegionObserver.java

```diff
@@ -211,9 +244,11 @@ private BatchMutateContext(int clientVersion) {
   private long slowIndexPrepareThreshold;
   private long slowPreIncrementThreshold;
   private int rowLockWaitDuration;
+  private int concurrentMutationWaitDuration;
   private String dataTableName;
 
   private static final int DEFAULT_ROWLOCK_WAIT_DURATION = 30000;
+  private static final int DEFAULT_CONCURRENT_MUTATION_WAIT_DURATION_IN_MS = 1000;
```

Comment:
1s seems long

Reply:
I will change it to 100ms and let me know if you have a different suggestion